input.php

```diff
@@ -2,14 +2,14 @@
     <head></head>
     <body>
         <form action='hidden.php' method='POST'>
-            <input type='text' name='name' />
-            <label for="name">What's their name?</label><p>
-            <input type='text' name='points' />
-            <label for="points">How many points?</label><p>
-            <input type='text' name='games' />
-            <label for="games">How many games?</label><p>
-            <input type='text' name='rings' />
-            <label for="rings">How many rings?</label><p>
+            <p><input type='text' name='name' />
+            <label id="name">What's their name?</label></p>
```

Comment:
sorry this isnt right. i probably did a bad job of explaining it. take a look at the example here:
https://www.w3schools.com/tags/tag_label.asp
inparticular, read the "tips and notes" section